packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/MultiShareBackupModal/MultiShareBackupStep5Done.tsx

```diff
@@ -0,0 +1,135 @@
+import { Card, Row, Icon, Column, Text } from '@trezor/components';
+
+import { Translation } from 'src/components/suite';
+import { Body, Section } from './MultiShareModalLayout';
+import { borders, spacings, spacingsPx } from '@trezor/theme';
+import { ReactNode } from 'react';
+import { TranslationKey } from '@suite-common/intl-types';
+import styled from 'styled-components';
+
+const GradientCallout = styled.div`
+    background-image: linear-gradient(
+        to right,
+        ${({ theme }) => theme.backgroundPrimarySubtleOnElevation1},
+        ${({ theme }) => theme.backgroundAlertYellowSubtleOnElevation1}
+    );
+
+    border-radius: ${borders.radii.xxs};
+    width: 100%;
+`;
+
+const GradientCalloutCard = styled.div`
+    flex: 1;
+    padding: ${spacingsPx.lg} ${spacingsPx.md};
+`;
+
+const IconQuestionMarkWrapper = styled.div`
+    position: relative;
+    margin-bottom: ${spacingsPx.xxxl};
+`;
+
+const IconQuestionMark = styled(Icon)`
+    position: absolute;
+    top: -7px;
+    left: 25px;
+`;
+
+const TextDiv = styled(Text)`
```

Comment:
This could have been a `Paragraph` instead.

Reply:
Not sure what is proper way of using it.

Paragraph has `// @TODO should be implemented with Text component: <Text as="p">...</Text>` comment. :man_shrugging:

Reply:
I'd use Paragraph so that it can be refactored later when we get to the TODO. But not necessary.